### DISCUSSION TITLE
2022/11/01/building-a-url-shortener-in-12-lines-of-code-using-cloudflare-workers/

### DISCUSSION THREAD

**Discussion by giscus** [thomaslevesque/blog#110] [Blog comments]
# 2022/11/01/building-a-url-shortener-in-12-lines-of-code-using-cloudflare-workers/

https://thomaslevesque.com/2022/11/01/building-a-url-shortener-in-12-lines-of-code-using-cloudflare-workers/

**Comment by miantiao-me** [comment]
I have also created a URL shortening tool with visit statistics functionality.

https://github.com/ccbikai/sink
